src/Avalonia.Diagnostics/Diagnostics/DevToolsViews.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Avalonia.Diagnostics;
+
+/// <summary>
+/// Diagnostic views available in DevTools
+/// </summary>
+public enum DevToolsViews
```

Comment:
Naming: only flags enum are usually plural.
This probably should be `DevToolsViewType/Kind` or something similar.